specification/dataprotection/resource-manager/Microsoft.DataProtection/stable/2023-01-01/dataprotection.json

```diff
@@ -0,0 +1,7082 @@
+        "includedResourceTypes": {
+          "description": "Gets or sets the include resource types property. This property sets the resource types to be included during restore.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "excludedResourceTypes": {
+          "description": "Gets or sets the exclude resource types property. This property sets the resource types to be excluded during restore.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "labelSelectors": {
+          "description": "Gets or sets the LabelSelectors property. This property sets the resource with such label selectors to be included during restore.",
+          "items": {
+            "type": "string"
+          },
+          "type": "array"
+        },
+        "persistentVolumeRestoreMode": {
+          "description": "Gets or sets the PV Restore Mode property. This property sets whether volumes needs to be restored.",
+          "enum": [
+            "RestoreWithVolumeData",
+            "RestoreWithoutVolumeData"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PersistentVolumeRestoreMode",
+            "modelAsString": true
+          }
+        },
+        "conflictPolicy": {
+          "description": "Gets or sets the Conflict Policy property. This property sets policy during conflict of resources during restore.",
+          "enum": [
+            "Skip",
+            "Patch"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ExistingResourcePolicy",
+            "modelAsString": true
+          }
+        },
+        "namespaceMappings": {
+          "description": "Gets or sets the Namespace Mappings property. This property sets if namespace needs to be change during restore.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        }
+      },
+      "x-ms-discriminator-value": "KubernetesClusterRestoreCriteria"
+    },
+    "RecoveryPointDataStoreDetails": {
+      "description": "RecoveryPoint datastore details",
+      "properties": {
+        "creationTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "expiryTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "id": {
+          "type": "string"
+        },
+        "metaData": {
+          "type": "string"
+        },
+        "state": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string"
+        },
+        "visible": {
+          "type": "boolean"
+        },
+        "rehydrationExpiryTime": {
+          "format": "date-time",
+          "type": "string",
+          "readOnly": true
+        },
+        "rehydrationStatus": {
+          "readOnly": true,
+          "enum": [
+            "CREATE_IN_PROGRESS",
```

Comment:
These don't conform to normal Azure API style, which is to use PascalCase enumeration names. I'm not sure whether it is worth fixing this in terms of *how breaking* it will be to preview customers. Please consider whether you want to fix this or not, and seek input from Breaking Changes Reviewers as needed.

Reply:
This is the breaking change in our platform, and we will consider it internally within our platform team later in future.